frontend/src/App.js

```diff
@@ -26,6 +27,24 @@ const useStyles = makeStyles((theme) => ({
  */
 export default function App() {
   const classes = useStyles();
+  const matchDataRefreshRate = 5000;
+  const [matchStatus, setMatchStatus] = React.useState('Unmatched');
+
+  /** Initiate GET request to search-match servlet */
+  function getMatch() {
+    axios.get('/api/v1/search-match')
+        .then((response) => {
+          console.log(response);
+          if (response.status === 200 && response.data.matchStatus === 'true') {
+            setMatchStatus('Matched');
+            clearInterval(interval);
+          }
+        })
+        .catch((error) => {
+          console.log(error);
```

Comment:
What does the user see if this happens? Should we show an error?

Reply:
Currently, added an alert to the screen along with a TO-DO (linked to an issue) to potentially add a page for that